backend/song/tests/test_views.py

```diff
@@ -0,0 +1,451 @@
+from rest_framework import status
```

Comment:
what do you think about the length of this file?
the number of lines in it?
can't we find a way to avoid having larges files?